docs/includes/apiargs-common-option.yaml

```diff
@@ -10,6 +10,13 @@ operation: ~
 optional: true
 ---
 arg_name: option
+name: maxAwaitTimeMS
+type: integer
+description: |
+    Positive integer denoting the time limit in milliseconds for the server to block
+    a getMore operation if no data is available. This option should only be used in
+    conjunction with the "tailable" and "awaitData" options.
```

Comment:
This can also be changed to "This option should only be used if ``cursorType`` is ``TAILABLE_AWAIT``".

Reply:
Also, if we're no longer sharing the option definition outside of `apiargs-MongoDBCollection-method-find-option.yaml`, we can just put this that file.